config_script.py

```diff
@@ -0,0 +1,55 @@
+"""
+Program that converts a fluentd config file to a master agent config file.
+
+Usage:
+    python3 -m config_script <fluentd path> <master path>
+Where:
+    master path: directory to store master agent config file in
+    fluentd path: path to the fluentd config file
+"""
+
+import os
+import subprocess
+import sys
+
+
+def read_file(path: str) -> str:
+    """Reads contents of file at path."""
+    with open(path, 'rt') as f:
+        return f.read()
+
+
+def get_object(args: list) -> None:
+    """Run ruby exec file and get message object."""
+    try:
+        subprocess.run(['config_converter/config_parser/bin/config_parser'] +
+                       args,
+                       check=True)
+    except subprocess.CalledProcessError:
+        sys.exit()
+    if not os.path.exists(args[-1] + '/config.json'):
+        sys.exit()
+
+
+def convert_object(args: list) -> None:
+    """Run config_mapper python file to create yaml file."""
+    config_json: str = read_file(f'{args[0]}/config.json')
+    cli_args = args + [config_json]
+    try:
+        subprocess.run([
+            'python3', '-B', '-m',
+            'config_converter.config_mapper.config_mapper'
+        ] + cli_args,
+                       check=True)
+    except subprocess.CalledProcessError:
+        sys.exit()
+
+
+if __name__ == '__main__':
+    args_passed = sys.argv[1:]
```

Comment:
I think you just do:

    fluentd_path, master_dir = sys.argv[1], sys.argv[2]

similarly to above.

Reply:
That would be assuming the arguments passed are correct in number. So instead I just pass all the arguments given to the Ruby code which parses the arguments correctly. I ideally want to parse the arguments in the main file so will be using argparse in my next PR, and this section will be written more neatly. Is that okay?